### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Customize New Project_Keyboard: After checking "Override" checkbox, focus is going to next control instead of going to enabled text-box 

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3317]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.4.3; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.4.3

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Tab till "Override" checkbox after "User Setting Files" text-box and press space to check it
13) Press tab to check if the focus is going to the enable text-box

**Actual Result:**
After checking "Override" checkbox, focus is going to next control instead of going to enabled text-box 

**Expected Result:**
After checking "Override" checkbox, focus should go to the enabled text-box instead of next control

**User Impact:**
Keyboard only users and screen reader users will find it difficult to access the page

**MAS Reference:**
[MAS 2.4.3](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})
[Bug_2_IntelliJ_Customize_Focus Order.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451121/Bug_2_IntelliJ_Customize_Focus.Order.pptx)

**Comment by Flanker32** [comment]
This view belong to IntelliJ which is out of the scope of Azure Toolkits, we have created an issue in jetbrains to track this tooptip, here is the link https://youtrack.jetbrains.com/issue/IDEA-221785

**Comment by Flanker32** [comment]
Importance of experience: Low, it's not a common case to override user settings file or local repository.
Quality of experience: Low, users could reach the target by tab all items around.
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
> This view belong to IntelliJ which is out of the scope of Azure Toolkits, we have created an issue in jetbrains to track this tooptip, here is the link https://youtrack.jetbrains.com/issue/IDEA-221785

As IntelliJ community replies, users could use `Shift` + `Tab` to get back to previous text field, here is the screen shot.

![3317-2](https://user-images.githubusercontent.com/12445236/64138388-700a0380-ce2f-11e9-9ee7-b2b9a300855a.gif)

**Comment by Flanker32** [comment]
@soumyatippapuram Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us verify it.

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749755
